main/migrations/0007_auto_20151115_1833.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
```

Comment:
But if I merge as is, I think will loose my property data in the first migration and then an empty data will be recreated.
Maybe the simplest approach is to squash your commits into a single commit containing 0 or 1 migration file ?